### PR TITLE
stage 6

### PR DESCRIPTION
**Pull request by ghost**
ports https://github.com/tgstation/-tg-station/pull/9847

### CODE REVIEW

**Comment by Nokington** [comment]
It needs a nerf I think. Getting set on fire within 25 tiles of it is a bit much.

**Comment by TDcoolguy300** [comment]
Current issue with this port, this is referring to the supermatter_Shard object, and not VG's supermatter object, needs to be fixed at
"if(istype(A, /obj/machinery/power/supermatter_shard) && !consumedSupermatter)" 
While the supermatter we use is" /obj/machinery/power/supermatter" and the shard subtype is "/obj/machinery/power/supermatter/shard"

**Comment by ghost** [comment]
oopsies.

**Comment by Nokington** [comment]
This still needs a nerf from 25 tiles. Thats just too much for people to random burst into flames

**Comment by collinxchu** [comment]
To be honest at 25 tiles you're being sucked towards it anyway.

I don't think it needs a nerf but the fact that both engines are so close together does make it a concern. Maybe a map edit is in order.

**Comment by ghost** [comment]
Nerfed to 10.

**Comment by Nokington** [comment]
@collinxchu A map edit would be nice yes but it just won't happen because the map is designed to just be a death trap. Why do you think the grav gen is in the middle of both the engines.

**Comment by ghost** [comment]
@Nokington merge ready, pls merge.

**Comment by Nokington** [comment]
@Chnkr I'm passing it by the heads first.

**Comment by Nokington** [comment]
My opinion on this has changed to do not want because this would most likely happen every round. It probably never happens on TG because I believe the supermatter shard is only ordered from cargo. But for us we just have a supermatter on station and odds are that it is gonna happen everytime the singulo is released. I would however be fine with this if we remap engineering so it isn't such a deathtrap.

**Comment by ghost** [comment]
the station is meant to be a fukken deathtrap, the chance of the singulo hitting it, is low since its `rng` based, i'd say we try this out for a week.

**Comment by AliiTea** [comment]
@Chnkr Great idea

File: code/modules/power/singularity/singularity.dm [hunk 1]
```diff
@@ -349,6 +374,14 @@
 	return
 
 
+/obj/singularity/proc/combust_mobs()
+	for(var/mob/living/carbon/C in orange(10, src))
+		C.visible_message("<span class='warning'>[C]'s skin bursts into flame!</span>", \
+						  "<span class='boldannounce'>You feel an inner fire as your skin is suddenly covered in fire!</span>")
```
Comment (nalarmg) [inline code/modules/power/singularity/singularity.dm:380]: This is a bit of an awkward sentence to read. 

"You feel your insides boil as your skin is engulfed in flame!" would be a better alternative.

**Comment by Nokington** [comment]
This isn't getting anywhere and none of the heads want it because it will happen every round until the SM is removed.

**Pull request closed without merging**